patch/kernel/archive/rockchip64-5.10/linux-90117-add-rk3399-roc-pc-plus-sound.patch

```diff
@@ -0,0 +1,176 @@
+--- a/sound/soc/codecs/es8328.c
```

Comment:
This change is for the sake of rk3399-roc-pc-plus but what you are truly doing is enhancing es8328 driver.
IMHO It should be reflected in the patch name as `general-enhance/backport-es8328-driver.patch`